packages/common/src/initiatives/_internal/InitiativeBusinessRules.ts

```diff
@@ -80,34 +81,39 @@ export const InitiativePermissionPolicies: IPermissionPolicy[] = [
     permission: "hub:initiative:discussions",
     dependencies: ["hub:initiative:view"],
   },
+  {
+    permission: "hub:initiative:workspace",
+    dependencies: ["hub:feature:workspace"],
+    environments: ["devext", "qaext"],
```

Comment:
is adding environments here redundant since they're on the `hub:feature:workspace` system permission policy?

Reply:
It is, but I wanted to limit my changes to "adding" vs changing... Teams should be curating these permissions as we move towards release, and we may push gating down from `hub:feature:workspace` to individual panes.